biz.aQute.bndlib/src/aQute/bnd/annotation/metatype/Configurable.java

```diff
@@ -1,14 +1,39 @@
 package aQute.bnd.annotation.metatype;
 
-import java.lang.reflect.*;
-import java.util.*;
-import java.util.regex.*;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Array;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.GenericArrayType;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Proxy;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.SortedSet;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import java.util.regex.Pattern;
 
 @SuppressWarnings({
 		"unchecked", "rawtypes"
 })
 public class Configurable<T> {
 	public static Pattern SPLITTER_P = Pattern.compile("(?<!\\\\)\\|");
+	public static final String	BND_ANNOTATION_CLASS_NAME	= "aQute.bnd.osgi.Annotation";
+	public static final String	BND_ANNOTATION_METHOD_NAME	= "getAnnotation";
```

Comment:
These don't need to be public, so then there is no new "API" and no need to bump the package minor version.